src/autosuggest/internal.tsx

```diff
@@ -226,7 +226,7 @@ const InternalAutosuggest = React.forwardRef((props: InternalAutosuggestProps, r
         stretchWidth={!__dropdownWidth}
         trigger={
           <InternalInput
-            type="search"
+            type="text"
```

Comment:
If we really have no choice but to use the `type="text"` attribute, that's fine. However, we will need to adjust `InternalInput` to keep the current search-type visuals and behavior. Otherwise this is a major regression.

Reply:
@jorycunningham and I have discussed offline, the proposal is to add a new **internal** type to make the input visually search but use type="text".